staff/ana-sanjuan/js-arrays/join.js

```diff
@@ -0,0 +1,10 @@
+
+// create a function that works as Array.prototype.join
+
+// // join.js
+
+function join(arr, separator) {
```

Comment:
Uau, all the code in one line. I´m impressed!. I suggest to add that if the separator is not defined use ",", like the original join.

Reply:
Thank you! :) 
regarding your suggestion, sorry I don´t understand it... what you mean?

Reply:
Done! :)